project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -253,6 +253,11 @@ function c4m_features_og_group_dashboard_block_info() {
     'cache' => DRUPAL_CACHE_PER_PAGE,
   );
 
+  $blocks['c4m_follow_block'] = array(
+    'info' => t('C4M Follow group block'),
```

Comment:
@ordavidil I've updated the title to say "C4M Follow group or project block" as this block can be used for both content types.